depends/common/fsuae/fsuae.txt

```diff
@@ -1 +1 @@
-fsuae https://github.com/kodi-game/libretro-fsuae/archive/db00e95fc7dcbe29f16926ec749e4693136a5c22.tar.gz
+fsuae https://github.com/kodi-game/libretro-fsuae/archive/75a94d0dfc86c9dbeb71ff47a12e1fb7ed4045e3.tar.gz
```

Comment:
This will get overwritten, is it indended?

Reply:
The initial change (only CMakeLitst.txt) got the build process a bit further.
So I then tried if a more recent version changes anything. As it still doesn't build it doesn't matter.

Reply:
Give up, we don't care about the fsuae core.